pkg/apis/config/v1alpha1/types.go

```diff
@@ -60,6 +60,9 @@ type Server struct {
 	Address string `json:"address,omitempty"`
 	// TLS contains the TLS configuration for the server.
 	TLS TLS `json:"tls"`
+	// MetricsPort is the port that the server will listen on for metrics.
+	// +optional
+	MetricsPort uint `json:"metricsPort,omitempty"`
```

Comment:
I know this is not served API and we already used uint for the `Port` field, but it is recommended by [API convetion](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types) to use int32 (or int64 if larger numbers are expected, which is not the case).